src/DeepFace.php

```diff
@@ -65,22 +68,29 @@ public function verify(
             throw new InvalidArgumentException("The path [{$img2_path}] for image#2 is not a file.");
         }
 
-        $output = $this->run(
-            filepath: __DIR__.'/../scripts/verify.py',
-            data: [
-                '{{img1_path}}' => $img1->getRealPath(),
-                '{{img2_path}}' => $img2->getRealPath(),
-                '{{enforce_detection}}' => $enforce_detection ? 'True' : 'False',
-                '{{align}}' => $align ? 'True' : 'False',
-                '{{model_name}}' => $model_name->value,
-                '{{detector_backend}}' => $detector_backend->value,
-                '{{distance_metric}}' => $distance_metric->value,
-                '{{normalization}}' => $normalization->value,
-            ],
-        );
+        try{
+            $output = $this->run(
+                filepath: __DIR__.'/../scripts/verify.py',
+                data: [
+                    '{{img1_path}}' => str_replace('\\', '/', $img1->getRealPath()),
+                    '{{img2_path}}' => str_replace('\\', '/', $img2->getRealPath()),
+                    '{{anti_spoofing}}' => $anti_spoofing ? 'True' : 'False',
+                    '{{enforce_detection}}' => $enforce_detection ? 'True' : 'False',
+                    '{{anti_spoofing}}' => $anti_spoofing ? 'True' : 'False',
+                    '{{align}}' => $align ? 'True' : 'False',
+                    '{{model_name}}' => $model_name->value,
+                    '{{detector_backend}}' => $detector_backend->value,
+                    '{{distance_metric}}' => $distance_metric->value,
+                    '{{normalization}}' => $normalization->value,
+                ],
+            );
+        } catch(DeepFaceException $e){
+            // if any of these images fails the spoof detection, it will throw 'Exception while processing imgX_path'
+            return array("error" => $e->getMessage());
```

Comment:
The `verify` method would throw `Exception while processing imgX_path` if any of the images is spoofed, since it uses `extract_faces` when verifying, and since the function returns a `VerifyResult`, I added array as a return to it to catch the exception message.

I can change it to any other way to return the exception or handle it in a different way if you have a better idea 😅

Reply:
I think best would be to let the exception just bubble up and do nothing about it in the package (for all methods). That way each user can decide what they do with that exception.

Reply:
Done!